src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -82,9 +83,9 @@ public bool IsContentModified
         }
 
         /// <summary>
-        /// Indicates whether or not the tag is self-closing.
+        /// The mode of the HTML element.
```

Comment:
Use comments more like those in `TagHelperExecutionContext` e.g. `Syntax of the element in the generated HTML.`